internal/data/k8spolicies/k8spolicies.go

```diff
@@ -72,7 +72,10 @@ func (r *k8spoliciesRepo) Update(ctx context.Context, model *biz.K8sPolicy, id s
 
 	if r.Eventer != nil {
 		producer, _ := r.Eventer.Lookup(identity, biz.ResourceType, model.ID)
-		evt := eventingapi.NewUpdateEvent(biz.ResourceType, model.Metadata.UpdatedAt, &eventingapi.EventResource[biz.K8sPolicyDetail]{
+		// Todo: model.Metadata.Reporters[0].LocalResourceId is the resourceId of the resource in the scope of the reporter.
+		// When we have multiple reporters we will need to make sure to publish the correct one in the eventing api.
```

Comment:
I think this is not an issue.

The entry point of this call is always:
https://github.com/project-kessel/inventory-api/blob/508fc28b4cc30ac7a5602cf2337263864f6bc968/api/kessel/inventory/v1beta1/resources/k8s_policies_service.proto#L22-L27

Which has a single reporter and resource data:
https://github.com/project-kessel/inventory-api/blob/508fc28b4cc30ac7a5602cf2337263864f6bc968/api/kessel/inventory/v1beta1/resources/k8s_policy.proto#L15-L23

In other words, `model.Metadata.Reporters` will always be a slice with just one element.

Reply:
Yes, but the idea is that multiple reporters can report the same resource. We will then aggregate in the database/orm and will eventually end with multiple reporters (in the future).
see: https://github.com/project-kessel/inventory-api/tree/main/api/kessel/inventory/v1beta1/documentation#resource-deduplication